src/.nuget/dir.targets

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+
+  <Target Name="VerifyVCRedist" BeforeTargets="GetSymbolPackageFiles" Condition="'$(_runtimeOSFamily)' == 'win'">
```

Comment:
@chcosta I think having the target before GetSymbolPackageFiles is the right thing, as that Target uses the NativeBinary Item, which depends on VCRedist files existing on the machine - we should fail before doing any operations on that data if VCRedist isn't present, as that work would've been unnecessary